.github/actions/deploy-backend/action.yml

```diff
@@ -74,8 +144,30 @@ runs:
           --tag ${{ env.ACR_NAME }}/${{ env.IMAGE_REPO }}:${{ env.TAG }}
       shell: bash
 
-    - name: Push Docker image
-      run: docker push ${{ env.ACR_NAME }}/${{ env.IMAGE_REPO }} --all-tags
+    - name: Push Docker image (signed)
+      if: env.USE_DCT == 'true'
+      run: |
+        echo "DOCKER_CONTENT_TRUST=1" >> $GITHUB_ENV
+        docker push ${{ env.ACR_NAME }}/${{ env.IMAGE_REPO }}:latest --disable-content-trust=false
+        docker push ${{ env.ACR_NAME }}/${{ env.IMAGE_REPO }}:${{ env.TAG }} --disable-content-trust=false
+        GREEN='\033[0;32m'
+        echo -e "${GREEN}#############" \
+        "\n${GREEN}#" \
+        "\n${GREEN}#   SIGNED" \
+        "\n${GREEN}#" \
+        "\n${GREEN}#############"
+      shell: bash
+
+    - name: Push Docker image (unsigned)
+      if: env.DOCKER_CONTENT_TRUST != 1
```

Comment:
should this use `env.USE_DCT == 'false'`? I'm concerned that both `env.USE_DCT == 'true'` and `env.DOCKER_CONTENT_TRUST != 1` could both be true, and we'll wind up overwriting the signed image with the unsigned one

Reply:
I was back and forth on this one. The issue was that `env.USE_DCT` wouldn't reliably be set to false, but I suppose i can use `env.USE_DCT != 'true'`. That should be fine!

 I did settle on the `env.DOCKER_CONTENT_TRUST != 1` since it is set to 1 within the signing step before it, but I will change to the `!= 'true'`